Italian/scappare/cercare-uscita.md

```diff
@@ -0,0 +1,6 @@
+Ti alzi per cercare una via d'uscita.
+Inciampi in una bottiglia rotta a metà, e ci sono vetri ovunque.
+
+[La raccogli](scappare/raccogli.md)
+[Cerchi una scopa](scappare/cerca-scopa.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.